### ISSUE TITLE
Not install under git-bash environment

### ISSUE THREAD

**Issue by fabriciofx** [llorllale/go-gitlint#48]
I've tried install `go-gitlint` in my environment (Windows 10, git-bash) and received this error message:
```
fbc@hades MINGW64 ~/Downloads
$ ./download-gitlint.sh
llorllale/go-gitlint crit uname_os_check 'MINGW64_NT-10.0' got converted to 'mingw64_nt-10.0' which is not a GOOS value. Please file bug at https://github.com/client9/shlib
```

**Comment by llorllale** [comment]
@fabriciofx sorry, `download-gitlint.sh` is a shell script meant to be run in UNIX-like environments (like your CI).

Workaround is to download the binary manually from [Releases](https://github.com/llorllale/go-gitlint/releases). Or you could provide suitable scripts in a PR.

**Comment by fabriciofx** [comment]
@llorllale Is it not supposed to work in `git-bash` environment? If not, consider this issue to put this information in README.

**Comment by llorllale** [comment]
@fabriciofx The supported OSes and architectures are implied in the binaries available in [Releases](https://github.com/llorllale/go-gitlint/releases). There is support for the common OSes: OSX, Windows, and "Linux". In this case, your shell is reporting "MINGW64_NT-10.0" as the OS which is not recognized and therefore it fails.
